lib/overcommit/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # Defines the gem version.
 module Overcommit
-  VERSION = '0.41.0'.freeze
+  VERSION = '0.42.0'.freeze
```

Comment:
Let's leave this out for now. We'll update the version when we release the next version. Thanks!